CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ description: |-
 ---
 
 # VKCS Provider's changelog
+#### v0.5.3 (unreleased)
+- Fixed error with creating large number of resources at once
```

Comment:
We should choose default time to write changes. For now we use both infinitive and past forms of verbs, looks strange. We discussed it some time ago and decided to use infinitives.

Reply:
Rewrote